Cargo.toml

```diff
@@ -26,7 +26,8 @@ members = [
 ]
 exclude = [
     "examples/*",
-    "swayfmt/test_macros"
+    "swayfmt/test_macros",
+    "forc-test/test_data"
```

Comment:
Just curious, is it best practice to always exclude all test data?

Reply:
There is a long discussion about cargo's default behaviour once it comes to non-package files and publishing [here](https://github.com/rust-lang/cargo/issues/2063). So I tend to exclude big artifacts like a built package ~mbs from the publishing step. 

There are some downsides to this as well, since we strip the test data from the published package, tools like [crater](https://github.com/rust-lang/crater) maintained by Rust core team cannot use our packages to discover regression in the rust compiler.